tests-e2e/cypress/integration/runs/rdp_main_header_spec.js

```diff
@@ -0,0 +1,501 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// ***************************************************************
+import {stubClipboard} from '../../utils';
+
+describe('runs > run details page > header', () => {
+    let testTeam;
+    let testUser;
+    let testViewerUser;
+    let testPublicPlaybook;
+    let playbookRun;
+
+    before(() => {
+        cy.apiInitSetup().then(({team, user}) => {
+            testTeam = team;
+            testUser = user;
+
+            // # Create another user in the same team
+            cy.apiCreateUser().then(({user: viewer}) => {
+                testViewerUser = viewer;
+                cy.apiAddUserToTeam(testTeam.id, testViewerUser.id);
+            });
+
+            // # Login as testUser
+            cy.apiLogin(testUser);
+
+            // # Create a public playbook
+            cy.apiCreatePlaybook({
+                teamId: testTeam.id,
+                title: 'Public Playbook',
+                memberIDs: [],
+            }).then((playbook) => {
+                testPublicPlaybook = playbook;
+            });
+        });
+    });
+
+    const openRunActionsModal = () => {
+        // # Click on the run actions modal button
+        cy.findByRole('button', {name: /Run Actions/i}).click({force: true});
+
+        // * Verify that the modal is shown
+        cy.findByRole('dialog', {name: /Run Actions/i}).should('exist');
+    };
+
+    const saveRunActionsModal = () => {
+        // # Click on the Save button without changing anything
+        cy.findByRole('button', {name: /Save/i}).click();
+
+        // * Verify that the modal is no longer there
+        cy.findByRole('dialog', {name: /Run Actions/i}).should('not.exist');
+    };
+
+    const getHeaderIcon = (selector) => {
+        return cy.findByTestId('run-header-section').find(selector);
+    };
+
+    const getDropdownItemByText = (text) => {
+        return cy.findByTestId('run-header-section').findByTestId('dropdownmenu').findByText(text);
+    };
+
+    describe('participant', () => {
```

Comment:
Nit: I think it's easier to read
```suggestion
    describe('as participant', () => {
```